### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by mishakoshilka**

### CODE REVIEW

File: src/convertToObject.js [hunk 1]
```diff
@@ -6,7 +6,29 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
```
Comment (volodymyr-soltys97) [inline src/convertToObject.js:11]:
```suggestion
```

File: src/convertToObject.js [hunk 2]
```diff
@@ -6,7 +6,29 @@
...
+
+  const result = {};
+  const splited = sourceString.split(';');
+
+  for (let i = 0; i < splited.length; i++) {
+    const target = splited[i].split(':');
```
Comment (volodymyr-soltys97) [inline src/convertToObject.js:17]:
```suggestion
    const [key, value] = splited[i].split(':').map(item => item.trim());
```

File: src/convertToObject.js [hunk 3]
```diff
@@ -6,7 +6,29 @@
...
 function convertToObject(sourceString) {
-  // write your code here
+  if (sourceString.length === 0) {
+    return {};
+  }
+
+  const result = {};
+  const splited = sourceString.split(';');
+
+  for (let i = 0; i < splited.length; i++) {
```
Comment (volodymyr-soltys97) [inline src/convertToObject.js:16]: Use a method forEach instead of for loop

File: src/convertToObject.js [hunk 4]
```diff
@@ -6,7 +6,29 @@
...
+    const target = splited[i].split(':');
+
+    if (target.length < 2) {
+      continue;
+    }
+
+    const key = target[0].trim();
+    const value = target[1].trim();
```
Comment (volodymyr-soltys97) [inline src/convertToObject.js:24]:
```suggestion
```

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's make your code better

**Review by anastasiiavorobiova — APPROVED** [review]
Great work!

**Pull request closed without merging**